src/style.css

```diff
@@ -0,0 +1,65 @@
+body {
+  display: flex;
+  margin: 0;
+
+  flex-wrap: wrap;
+
+}
+
+.box {
+  height: 300px;
+  flex-basis: 33.3%;
+  background-color: #f00;
+  font-family: "Arial", sans-serif;
+  font-size: 100px;
+  color: #fff;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.box:nth-child(2) {
+  background-color: #c00;
```

Comment:
You have classes like `box1`, `box2` etc. Why are you using `nth-child` instead of them?
Let's change it